COM-161 Add registration french translations

lauravignoli opened the pull request.

lauravignoli committed:
COM-161 Add registration french translations

prashantsable1991 commented:
@lauravignoli thanks you did a very nice job. But in a few places we need to add translations please find below images for references.
![COM-161](https://user-images.githubusercontent.com/47622485/55330997-07011700-54b0-11e9-85ef-7f7f94098c76.png)
![COM-161(2)](https://user-images.githubusercontent.com/47622485/55331002-08cada80-54b0-11e9-84ae-1d30de37915a.png)

lauravignoli committed:
COM-161 Enable gender translations on registration and dashboard

Updated existing method getPatientAttributeTypes that was retrieving genderText, ageText and bloodGroupText, and removed genderText from there, as it wasn't possible to translate when fetching it in that way.

Instead, used filter "gender" that retrieves the gender text from the global property mrs.genders.

lauravignoli committed:
COM-161 Fix unit test

Now that gender has been removed from patientAttributeTypes in patientProfile, it needs to be removed from the unit tests as well.
